package.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Add the `private: true` property.

Reply:
223fe958a669589d092187bd00f2f0d28b95007e